lib/lotus/view/template.rb

```diff
@@ -1,4 +1,9 @@
 require 'tilt'
+SUPPORT_ENGINE = %w(
+  haml
```

Comment:
I don't think we should include `haml`. There are many people out there don't use it. We let user explicitly require if needed